src/multipart.ts

```diff
@@ -71,15 +76,15 @@ const streamProcessor = (
   headers: RequestHeaders,
   callback: MultipartCallback
 ): void => {
-  const bufferArray: Array<Buffer> = [];
+  const bufferArray: Array<Uint8Array> = [];
   data.file.data
-    .on('data', (line: Buffer) => {
+    .on('data', (line: Uint8Array) => {
       bufferArray.push(line);
     })
     .once('end', () => {
-      // @ts-ignore
-      const bufferData: BufferedFile = Object.assign({}, data);
-      bufferData.file.data = Buffer.concat(bufferArray);
+      const bufferData: StreamingFile = Object.assign({}, data);
+      // @ts-ignore - bufferArray will not store Uint8Array's nested greater than depth 1
+      bufferData.file.data = bufferArray.flatMap((e) => e);
```

Comment:
Will the result here be Uint8Array? It might be Array<number> instead which is not what we want.

Reply:
Ah you're right, will fix